src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ListViewTests/UnoSamples_Tests.ListView.cs

```diff
@@ -119,5 +119,44 @@ public void Check_ListView_Swallows_Measure()
 			var finalMeasureCount = int.Parse(measureTextAfter);
 			Assert.AreEqual(initialMeasureCount, finalMeasureCount);
 		}
+
+		[Test]
+		[AutoRetry]
+		public void ListView_Weird_Measure_During_Arrange()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.ListView.ListView_Weird_Measure");
+
+			_app.WaitForText("StatusTextBlock", "Finished");
+
+			TakeScreenshot($"{nameof(ListView_Weird_Measure_During_Arrange)} after layout");
+
+			var heightStr = _app.GetText("HeightTextBlock");
+			var height = int.Parse(heightStr);
```

Comment:
You can use `_app.Query(q => q.Marked("dfdldks").First().Rect.Height` as well.

Reply:
Good point!

On Android will it be in physical or logical pixels?

Reply:
It should not matter. If you need something to be of a certain size, take one as a reference and compare.